deployment/https/readme.md

```diff
@@ -1,7 +1,39 @@
-This is configured for running on server for domain lj.starwit.de.
+This is configured for running on server for domain pb.starwit.de.
 
-* configure domain
-* login to server via ssh
-* **!!! set environment variables CLIENTID and CLIENTSECRET**
-* execute `init-letsencrypt.sh`
-* execute `docker-compose up -d`
\ No newline at end of file
+* 1. if you change domain configure / check domain in all files
+* 2. create server on hetzner
+* 3. create volume on named lj-mariadb
+* 4. login to server via ssh
+* 5. setup machine and install docker compose:
+
+```bash
+    apt update
+    apt install \
+        apt-transport-https \
+        ca-certificates \
+        curl \
+        gnupg-agent \
+        software-properties-common
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) \
+    stable"
+    apt update
+
+    # Install correct version of docker-compose
+    curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    chmod +x /usr/local/bin/docker-compose
+    ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+
+    apt install docker-ce docker-ce-cli containerd.io docker-compose
+```
```

Comment:
I think a link to the official setup would be best here. Otherwise you have to monitor and update this documentation all the time if docker compose comes out with a new version